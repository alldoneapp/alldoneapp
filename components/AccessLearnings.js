import React, { Component } from 'react'
import { StyleSheet, Text, View } from 'react-native'
import Icon from './Icon'

export default class AccessLearnings extends Component {
    constructor(props) {
        super(props)

        console.ignoredYellowBox = ['Setting a timer']
    }

    render() {
        return (
            <View style={styles.mainInfo}>
                <Icon name="grid" size={24} color="white"></Icon>
                <Text style={styles.itemText}>Access Learnings</Text>
            </View>
        )
    }
}

const styles = StyleSheet.create({
    mainInfo: {
        flexDirection: 'row',
        alignItems: 'center',
        width: 198,
        height: 40,
        borderRadius: 4,
        backgroundColor: '#0D55CF',
        borderColor: '#0D55CF',
        paddingLeft: 19,
    },
    itemText: {
        paddingLeft: 15,
        color: 'white',
        fontFamily: 'Roboto-Medium',
        fontSize: 14,
        lineHeight: 14,
        letterSpacing: 0.05,
    },
})
