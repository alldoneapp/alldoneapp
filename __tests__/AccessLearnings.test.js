import React from 'react'
import AccessLearnings from '../components/AccessLearnings'

import renderer from 'react-test-renderer'

describe('AccessLearning component', () => {
    describe('AccessLearning snapshot test', () => {
        it('should render correctly', () => {
            const tree = renderer.create(<AccessLearnings />).toJSON()
            expect(tree).toMatchSnapshot()
        })
    })
})
