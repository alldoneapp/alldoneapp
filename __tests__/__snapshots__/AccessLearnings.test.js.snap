// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccessLearning component AccessLearning snapshot test should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0D55CF",
      "borderColor": "#0D55CF",
      "borderRadius": 4,
      "flexDirection": "row",
      "height": 40,
      "paddingLeft": 19,
      "width": 198,
    }
  }
>
  <View>
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "alldone",
          "fontSize": 24,
          "opacity": 1,
        }
      }
    >
      î¦Ž
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "white",
        "fontFamily": "Roboto-Medium",
        "fontSize": 14,
        "letterSpacing": 0.05,
        "lineHeight": 14,
        "paddingLeft": 15,
      }
    }
  >
    Access Learnings
  </Text>
</View>
`;
